Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -154,7 +154,7 @@ private void OnChangeTemperature(EntityUid uid, GasThermoMachineComponent thermo
                 thermoMachine.TargetTemperature = MathF.Min(args.Temperature, thermoMachine.MaxTemperature);
             else
                 thermoMachine.TargetTemperature = MathF.Max(args.Temperature, thermoMachine.MinTemperature);
-            thermoMachine.TargetTemperature = MathF.Max(thermoMachine.TargetTemperature, Atmospherics.TCMB);
+            thermoMachine.TargetTemperature = MathF.Max(thermoMachine.TargetTemperature, 0.0f);
```

Comment:
probably should leave this at Atmospherics.TCMB or some minimal temp value because hellfire freezers can leak cold into the air and give you 0K gas
"wtf div0, server must die" is also a possibility

Reply:
changed